packages/admin/resources/views/widgets/account-widget.blade.php

```diff
@@ -15,14 +15,16 @@ class="w-10 h-10 rounded-full bg-gray-200 bg-cover bg-center"
                     {{ __('filament::widgets/account-widget.welcome', ['user' => \Filament\Facades\Filament::getUserName($user)]) }}
                 </h2>
 
-                <p class="text-sm">
+                <form method="POST" action="{{ route('filament.auth.logout') }}" class="text-sm">
+                    @csrf
+
                     <a href="{{ route('filament.auth.logout') }}" @class([
```

Comment:
Instead of using JavaScript to submit the form, surely we can just make this tag a `<button type="submit">`?

Reply:
I ended up going the same route as the breeze methods, but if you prefer a button I can swap it over.